### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by Rohitha-pudu**
Added home page ,when user clicks on view button it navigates to the satellite data page(station)

### CODE REVIEW

File: src/Home.jsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+      <style jsx>{`
+        
+        .div {
+          background-color: #000;
+          display: flex;
+          flex-direction: column;
+          font-size: 26px;
+          font-weight: 400;
+        }
+        .div-2 {
```
Comment (Witty-Wizard) [inline src/Home.jsx:48]: Move all these to a separate CSS file

File: src/Home.jsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
+
```
Comment (Witty-Wizard) [inline src/Home.jsx:1]: Add comments!

File: src/Home.jsx (hunk 1 above)
Comment (Rohitha-pudu) [reply src/Home.jsx:48]: ok

File: src/Home.jsx [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+          <div className="div-3">
+            <div>
+            <img src="gagan.png" alt="Logo" className="div-4"/>
+            </div>
+            <div className="div-5">
+              Empowering Exploration: Open Source Ground Station for Cansat
+              Satellites
+            </div>
+            <div>
+              <button className="div-6" onClick={Station}>
```
Comment (Witty-Wizard) [inline src/Home.jsx:32]: Use Button provided by Material UI, Check the ConnectButton component inside the Serial/UI directory

**Comment by Rohitha-pudu** [comment]
done @Witty-Wizard

File: src/Home.jsx (hunk 2 above)
Comment (Witty-Wizard) [reply src/Home.jsx]: @Rohitha-pudu , Please do this aswell

File: src/Home.jsx (hunk 3 above)
Comment (Witty-Wizard) [reply src/Home.jsx:38]: @Rohitha-pudu , This as well

File: src/Home.jsx (hunk 2 above)
Comment (Rohitha-pudu) [reply src/Home.jsx]: done it already @Witty-Wizard

File: src/Home.jsx (hunk 3 above)
Comment (Rohitha-pudu) [reply src/Home.jsx]: > @Rohitha-pudu , This as well

Done @Witty-Wizard

**Comment by Witty-Wizard** [comment]
@Rohitha-pudu, shift the PR to this branch https://github.com/Gagan-Space/Cansat-Ground-Station/tree/landing-page-frontend-ui

**Comment by Rohitha-pudu** [comment]
> @Rohitha-pudu, shift the PR to this branch https://github.com/Gagan-Space/Cansat-Ground-Station/tree/landing-page-frontend-ui

Done @Witty-Wizard

**Pull request closed without merging**